src/EventHandler/Message.php

```diff
@@ -25,6 +25,12 @@ abstract class Message extends AbstractMessage
     /** Content of the message */
     public readonly string $message;
 
+    private bool $reactionsCached = false;
```

Comment:
Could you please merge + make these protected? It's a new requirement for all properties in messages due to the new IPC structure.